quic/s2n-quic-platform/Cargo.toml

```diff
@@ -17,25 +17,25 @@ testing = ["std", "generator", "futures/std", "io-testing"] # Testing allows to
 io-testing = ["bach"]
 generator = ["bolero-generator", "s2n-quic-core/generator"]
 tokio-runtime = ["futures", "tokio"]
-xdp = ["s2n-quic-xdp"]
+xdp = ["dep:s2n-quic-xdp"]
```

Comment:
what does `dep:` do?

Reply:
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies - basically makes it explicit that it's enabling a dependency, not a feature. but i'm going to remove this for now and do a separate PR to fix the `--all-feature` stuff in CI